src/psyclone/dynamo0p3.py

```diff
@@ -53,10 +53,11 @@
 from psyclone import psyGen
 from psyclone.configuration import Config
 from psyclone.core import AccessType, Signature
+from psyclone.domain.common.psylayer import PSyLoop
```

Comment:
Not your fault for both, but `pylint` reports
```
dynamo0p3.py:51:0: W0611: Unused import fparser (unused-import)
dynamo0p3.py:56:0: W0611: Unused PSyLoop imported from psyclone.domain.common.psylayer (unused-import)
```
Would you mind fixing this?

Reply:
also fixed a load of super() without argumants from pylint while I was there
```
dynamo0p3.py:4355:8: R1725: Consider using Python 3 style super() without arguments
dynamo0p3.py:4510:8: R1725: Consider using Python 3 style super() without arguments
dynamo0p3.py:3462:8: R1725: Consider using Python 3 style super() without arguments
dynamo0p3.py:2636:8: R1725: Consider using Python 3 style super() without arguments
dynamo0p3.py:2019:8: R1725: Consider using Python 3 style super() without arguments
dynamo0p3.py:1786:8: R1725: Consider using Python 3 style super() without arguments 
dynamo0p3.py:510:8: R1725: Consider using Python 3 style super() without arguments
```

Reply:
Much appreciated, thank you!